template_example/dev/ansible/00_create_intention.yml

```diff
@@ -1,6 +1,10 @@
 - name: set variable
   set_fact:
-    consul_token: "master"
+    consul_token: "{{ lookup('env', 'consul_master_token') }}"
+
+- name: Print consul_token
+  debug:
+    msg: "Her's your token: '{{ consul_token }}'"
```

Comment:
```suggestion
    msg: "Here's your token: '{{ consul_token }}'"
```

Reply:
Oh, that was only for debuging - will be deleted 😅